### ISSUE TITLE
'Not all changes could be converted to suggestions'

### ISSUE THREAD

**Issue by joshspicer** [microsoft/vscode-pull-request-github#6173]
Testing #6160

I may be understanding what "commenting range" here is, but I presumed that this change in the attached video should be ok? 

https://github.com/user-attachments/assets/c4e1bef5-0c17-4e3c-8df1-c28f3e4cc692

**Comment by alexr00** [comment]
I don't any commenting ranges in the diff editor that you have open. Is that file part of the PR that y9ou currently have checked out?

**Comment by joshspicer** [comment]
Yes, it's part of a PR. In this example i've included the bottom status bar that shows this change is part of https://github.com/joshspicer/almost-empty/pull/6

I think i'm not understanding what constitutes a 'commenting range'? For example on line 4 here I made an edit (and commented on it), then ran the action from the SCM panel. 


![Image](https://github.com/user-attachments/assets/fd050dfd-2b25-466f-89dd-73f95f9c067a)

**Comment by alexr00** [comment]
A commenting range is wherever you see the commenting bar and can leave a comment. If you make a change that's within that range then it should work, with the caveat that if you add or remove a line at the edge of a range it might not work because of where GitHub defines the ranges.

It looks like you made the change within a commenting range. Can you share a link to the PR by chance?

**Comment by joshspicer** [comment]
Here's the PR: https://github.com/joshspicer/almost-empty/pull/6

**Comment by alexr00** [comment]
The issue is when the only changes are line insertions.

**Comment by joshspicer** [comment]
I am no longer seeing the option available:
![image](https://github.com/user-attachments/assets/47426342-3e8d-465c-9a0c-7458e98c3181)

**Comment by alexr00** [comment]
Verified with @joshspicer in a call.
